application/helpers/LayoutHelper.php

```diff
@@ -452,6 +452,27 @@ protected function beforeSideMenuRender(array $aData)
     }
 
     /**
+     * New Topbar
+     * @param $aData
+     */
+    public static function renderTopbar($aData) {
+
+        $oTopbarConfig = TopbarConfiguration::fromViewData($aData);
+
+        if (!empty($oTopbarConfig->leftSideView)) $aData['topBar']['leftSideContent'] = App()->getController()->renderPartial($oTopbarConfig->leftSideView, $aData, true);
+        if (!empty($oTopbarConfig->rightSideView)) $aData['topBar']['rightSideContent'] = App()->getController()->renderPartial($oTopbarConfig->rightSideView, $aData, true);
+
+        // Temporary fix before implementing the TopbarWidget
+        $aData['topBar']['name'] = $oTopbarConfig->viewName;
```

Comment:
Why not just send the topbar config object to the views? `$aData['topBar']['topbarConfig'] = $oTopbarConfig;`

Reply:
We will when we implement the widget. Didn't want to go to far on this intermediate step.

Reply:
Alright